Fixing two open issues and make ipam-migrator usable again

Absynth723 opened the pull request:
This fixes #3 and #4 and makes ipam-migrator usable again. Pretty straightforward one-line fixes which come from other contributors' comments.

Commit:
Change vrfs to False to disable errors (importing VRFs was never implemented)

Commit:
Order of arguments for netbox constructor was mixed up (fixes #4)

Commit:
README modiffication

Callum027 commented:
Hi @absynth47215 , thanks for your contribution.

It's probably obvious to anyone actually paying attention to this repository that I haven't worked on this in years, but since some people are trying to use it, I think I owe you guys an explanation.

I developed this as our company (Catalyst IT Limited) was still on phpIPAM and needed to move to NetBox, and I got it working just enough to get that done. Our policy is open source and therefore I released it under the GPL as we normally do, but I didn't actually have any intention of maintaining it long term.

I no longer work for Catalyst, and have not done any work with phpIPAM or NetBox in many years, so it's no longer worth my time to work on this repository. If anyone else is willing to maintain it going forward, they are more than welcome to by forking this repository.

As for this change to get `ipam-migrator` working again, I'm more than happy to merge this, but if possible, I'd like you to add a notice saying that this repository is no longer maintained (or if you'd like, I can make a follow up PR noting this).

Commit:
vlans -> vlan testweise

Absynth723 commented:
Hi, for some odd reason I didn't get notified on this. I'll fix up a bug that I might have accidentally introduced while trying to make the migrator work with current phpIPAM versions (#7), do some testing and modify the README to mark the repo as abandoned.
I'm in exactly the same situation as you, so I think it's fair to say that this tool might be one of them "get to work, use once, pass down to next guy" tools that might or might not be useful for others down the line. It certainly helped me a lot because it saved me from reinventing the wheel.

Commit:
Bugfix for #7 (re-commit a working version after testing)

Commit:
README modification as per Callum's request (abandonware)

Commit:
Update README.md

Commit:
Update README.md

Commit:
Remove debug output

Commit:
Merge branch 'master' of https://github.com/absynth47215/ipam-migrator

Commit:
Update README.md

Absynth723 commented:
I fixed up #7 and removed some debug output, and as far as I can tell, this is my working version that I used for my migration. I also changed the README.
